### DISCUSSION TITLE
How to Remove Banner for New UI

### DISCUSSION THREAD

**Discussion by PMcLeanGLX** [inventree/InvenTree#8493] [Q&A]
With the new UI there are issues with setting up email notifications, new users etc. 

How do I get rid of the banner asking me to switch to the new UI so the users I deploy don't accidentally click it?

**Comment by wolflu05** [comment]
To disable the pui banners referr to the configuration documentation: https://docs.inventree.org/en/stable/start/config/#customization-options 

But if you find things that aren't working, feel free to make an issue for them if there isn't one already so that PUI can get better. It is currently in beta phase and CUI will be removed in some time.
